package com.test2;

class MyClass extends Exception{
	
}


public class CustomExceptionTest {

	public static void main(String[] args) throws Exception {
	System.out.println("start");
	
	     //try {
			// throw new MyClass(); // exception 객체 생성. -> 아직은 그냥 instance.
		
		               // exception 객체를 던져야함 ; throw -> exception발생.
		//} catch(Exception e) {
			
		//}
		
		
		System.out.println("finish");
	}

}
